### ISSUE TITLE
live export error

### ISSUE THREAD

**Issue by cdaein** [raphaelameaume/fragment#42]
Hi,
thank you for the great project.

My issue is that I cannot run the live export website.

To reproduce,

1. create an empty project
```sh
fragment ./sketch.js --new
```
2. create a build
```sh
fragment ./sketch.js -b 
```
3. the build succeeds with terminal log:
```
[fragment] ℹ Building files for:
- sketch.js
vite v3.2.5 building for production...
✓ 195 modules transformed.
12:15:59 AM [vite-plugin-svelte] dom compile done.
package       	files	 time	  avg
fragment-tools	   68	0.65s	9.6ms
../../../../../../../../../Desktop/fragment-bug/sketch/assets/mp4.2ae3801e.wasm        41.00 KiB
../../../../../../../../../Desktop/fragment-bug/sketch/index.html                      0.62 KiB
../../../../../../../../../Desktop/fragment-bug/sketch/assets/MidiPanel.a1686b2e.js    2.63 KiB / gzip: 1.33 KiB
../../../../../../../../../Desktop/fragment-bug/sketch/assets/Exports.bcb82c63.js      4.57 KiB / gzip: 1.60 KiB
../../../../../../../../../Desktop/fragment-bug/sketch/assets/sketch.412731aa.js       0.22 KiB / gzip: 0.16 KiB
../../../../../../../../../Desktop/fragment-bug/sketch/assets/2DRenderer.8ac322c0.js   0.08 KiB / gzip: 0.09 KiB
../../../../../../../../../Desktop/fragment-bug/sketch/assets/Console.bb8de45e.css     6.37 KiB / gzip: 1.75 KiB
../../../../../../../../../Desktop/fragment-bug/sketch/assets/index.a53bd8f9.css       23.83 KiB / gzip: 4.21 KiB
../../../../../../../../../Desktop/fragment-bug/sketch/assets/Console.cff4120d.js      52.21 KiB / gzip: 12.27 KiB
../../../../../../../../../Desktop/fragment-bug/sketch/assets/index.ed224408.js        198.13 KiB / gzip: 64.08 KiB
[fragment] ✔ Built files for:
- sketch.js
```

4. run the live version (in my case, using VS Code live server), then get an error in the browser console:
```
Uncaught TypeError: Cannot read properties of undefined (reading 'buildConfig')
    at n.$$.update (index.f22b538f.js:98:80358)
    at se (index.f22b538f.js:1:8784)
    at new io (index.f22b538f.js:98:80779)
    at Array.Ph (index.f22b538f.js:98:88622)
    at Array.Uh (index.f22b538f.js:98:90443)
    at Pe (index.f22b538f.js:1:1646)
    at Array.Ih (index.f22b538f.js:98:87522)
    at Pe (index.f22b538f.js:1:1646)
    at Array.wh (index.f22b538f.js:98:81230)
    at Ah (index.f22b538f.js:98:84286)
```

5. in the browser window, it's stuck with the on-screen message:
```
[fragment]
loading dependencies...
```

thank you for any help!

**Comment by raphaelameaume** [comment]
Hello @cdaein. This was fixed in (#29) and released in v0.1.12. Apologies for getting back to you so late.

**Comment by cdaein** [comment]
I was going to test it out, but when I create a new sketch with `fragment` command, it doesn't show the active port:

```sh
❯ fragment ./sketch.js --new --template=2d
[fragment] Created sketch.js on disk.
[fragment] ℹ Building files for:
- sketch.js
[fragment] ℹ Starting server...
[fragment] ✔ Server started at:
```

**Comment by raphaelameaume** [comment]
Indeed, the fix was pushed after the release of v0.1.12. I fixed it in #45 and released it in #48. If you install the latest version, the server URLs should be back.

**Comment by cdaein** [comment]
I can confirm it works. Thank you for the fix.
